types/carbon__pictograms-react/lib/storage--product/index.d.ts

```diff
@@ -0,0 +1 @@
+export { StorageProduct as default } from "../../";
```

Comment:
The package exports these using `export=`
https://unpkg.com/browse/@carbon/pictograms-react@11.2.0/lib/storage--product/index.js

This needs `export =` instead of `export default`. Refer to https://github.com/DefinitelyTyped/DefinitelyTyped#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default

Reply:
Updated the lib declaration files to use `export =` syntax